### DISCUSSION TITLE
How to turn on Top Up mode (it's constantly greyed out)?

### DISCUSSION THREAD

**Discussion by marcinlesek** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#757] [Q&A] [answered]
Hello!

Firstly, AlDente is fantastic - thank everyone involved with this app's creation. 🚀 

I come here to ask about using **Top Up** mode. I have a charge limit of 75% because I mainly work stationary with my MacBook Pro (2018 i9). Sometimes, I need to grab it and go outside, so I want to charge my battery fully. Unfortunately, the Top Up button is **constantly greyed out** so I can't use it.

<img width="472" alt="Screen Shot 2022-08-19 at 11 22 49" src="https://user-images.githubusercontent.com/13199192/185590340-89dcc365-08d3-4794-b73a-61cb36168bac.png">

What I'm doing wrong? I thought that maybe shell mode (having MacBook with a closed lid when working can do that), but I tried in both (working with a closed lid and only on my MacBook without any external monitors), and it's still not working. 

I'm using free AlDente.

Thanks for any tips or comments!

Best,
Marcin

**Comment by MatthiasKerbl** [comment, marked as answer]
Hi @marcinlesek,

I am glad you like our little tool AlDente so much.
Unfortunately, the Top Up feature is only available in AlDente Pro which you can buy on our website here: https://apphousekitchen.com/pricing/

Cheers,
Matthias

**Comment by marcinlesek** [comment]
Ah, you're right. I don't know why I was checking it and don't see that `Top Up` feature is paid. Thank you for explanation! 🙌🏻
